src/content/docs/apm/agents/java-agent/troubleshooting/jboss-and-wildfly-incomplete-telemetry.mdx

```diff
@@ -11,25 +11,26 @@ redirects:
   - /docs/agents/java-agent/troubleshooting/jboss-and-wildfly-incomplete-telemetry
 ---
 
-## Problem
+## Problem [#problem]
 
 You are getting incomplete telemetry from an app using JBoss EAP 7.4 (or up) or Wildfly 23 (or up). 
-This could be in the form of web requests being translated into non-web transactions or your dispatcher name is incorrect.
+This can happen in two ways:
```

Comment:
There are more ways the telemetry can be incomplete. I'd put them as possible examples.